vulcanai/models/metrics.py

```diff
@@ -243,7 +243,7 @@ def get_specificity(targets, predictions, average=None):
         specificity = tn / (tn + fp)
 
         # TODO: implement other options
-        if average == "macro":
+        if average == "macro" or average == "binary":
             specificity = np.average(specificity)
```

Comment:
I don't think it's a good idea to average a binary sensitivity or specificity since really it's only referring to the positive class you are trying to predict. @nirtiac @joemehltretter is there a way to standardize which class is the positive one and in the binary case return the corresponding value of the positive class?

Reply:
Hmmmmm ok yeah. However, I'm reluctant to standardize unless we make it absolutely bulletproof as I see that resulting in uncaught errors with serious implications. We could simply just return None for those cases.

Reply:
yeah I can see that being problematic and harder to use. perhaps there shouldn't be any averaging for binary cases?

Reply:
As a counterpoint to myself, take a look at what scikit does for recall score (I'm emulating skl_metrics as much as possible in this class). 
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html

Reply:
I do like the pos_label quality if an only if binary is used. + lots of error checking and verbosity in the documentation